snf-image-host/backends/dst/rbd/rbd.in

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
`rbd.in` should not be executable. It's an template file for a script. Not a script itself.